hook.js

```diff
@@ -20,6 +23,12 @@ if (NODE_MAJOR >= 20) {
   getExports = (url) => import(url).then(Object.keys)
 }
 
+if (NODE_MAJOR >= 16) {
```

Comment:
Both `getEsmExports` and `getExports` need to have a variable declared with `let`, like `entrypoint` is on line 18.